packages/monorepo-builder/packages/Testing/ComposerJson/ComposerJsonSymlinker.php

```diff
@@ -39,31 +40,50 @@ final class ComposerJsonSymlinker
      */
     private $packagePathResolver;
 
-    public function __construct(ComposerJsonProvider $composerJsonProvider, PackagePathResolver $packagePathResolver)
-    {
+    /**
+     * @var JsonFileManager
+     */
+    private $jsonFileManager;
+
+    public function __construct(
+        ComposerJsonProvider $composerJsonProvider,
+        PackagePathResolver $packagePathResolver,
+        JsonFileManager $jsonFileManager,
```

Comment:
@leoloso please remove `,` after `$jsonFileManager`: 

```suggestion
        JsonFileManager $jsonFileManager
```

Reply:
Thanks! Fixed now